README.md

```diff
@@ -0,0 +1,81 @@
+# Sumo Logic OpenTelemetry auto-instrumentation for JavaScript
+
+The [Sumo Logic](https://www.sumologic.com/) OpenTelemetry auto-instrumentation for JavaScript library enables tracing in the browser.
+
+## Installation
+
+The easiest way to start collecting traces from yor website is to put the code below inside the `<head></head>` tags on your website:
+
+```html
+<script
+  src="https://cdn-nite-www.sumologic.net/ui/js/thirdparty/sumologic-opentelemetry.js"
+  data-collection-source-url="sumo_logic_traces_collector_source_url"
+  data-service-name="name_of_your_web_application"
+>
+```
+
+See [functionalities](#Functionalities) for informations about the script size and [configuration](#Configuration) for all supported options.
+
+There are no other required actions needed to take. With properly provided `data-collection-source-url` and `data-service-name` your website is ready and will send collected traces to the specified Sumo Logic collector.
+
+## Manual installation
+
+The other option is to bundle this library inside your project and initialize it.
+
+Inside your project directory execute `npm install @sumologic/opentelemetry-tracing`.
+
+Tracing needs to be initialized preferably before other functionalities in your code:
+
+```javascript
+import { initializeTracing } from "@sumologic/opentelemetry-tracing";
+
+initializeTracing({
+  collectionSourceUrl: "sumo_logic_traces_collector_source_url",
+  serviceName: "name_of_your_web_application",
+});
+```
+
+## Functionalities
+
+This library contains built-in OpenTelemetry packages:
+
+- [@opentelemetry/core](https://www.npmjs.com/package/@opentelemetry/core)
+- [@opentelemetry/tracing](https://www.npmjs.com/package/@opentelemetry/tracing)
+- [@opentelemetry/web](https://www.npmjs.com/package/@opentelemetry/web)
+- [@opentelemetry/plugin-xml-http-request](https://www.npmjs.com/package/@opentelemetry/plugin-xml-http-request)
+- [@opentelemetry/context-zone](https://www.npmjs.com/package/@opentelemetry/context-zone)
+- [@opentelemetry/plugin-document-load](https://www.npmjs.com/package/@opentelemetry/plugin-document-load)
+- [@opentelemetry/plugin-user-interaction](https://www.npmjs.com/package/@opentelemetry/plugin-user-interaction)
+
+See [@opentelemetry/plugin-xml-http-request](https://www.npmjs.com/package/@opentelemetry/plugin-xml-http-request), [@opentelemetry/plugin-document-load](https://www.npmjs.com/package/@opentelemetry/plugin-document-load) and [@opentelemetry/plugin-user-interaction](https://www.npmjs.com/package/@opentelemetry/plugin-user-interaction) for more details about auto-instrumented functionalities.
+
+To connect your traces with backend operations, make sure you support [W3C Trace Context](https://www.w3.org/TR/trace-context/) HTTP headers.
+
+The script size is 92KB.
+
+## Configuration
+
+Both `<script />` tag and manual installation can be configured with following parameters:
```

Comment:
This is XML syntax which doesn't work in HTML. Just `script tag` should be enough, optionally `` `<script>` tag ``.